Java/PlatinumGenomes-variant-transformation/README.md

```diff
@@ -68,6 +68,7 @@ java -cp target/non-variant-segment-transformer-*runnable.jar \
   com.google.cloud.genomics.examples.TransformNonVariantSegmentData \
   --project=YOUR_GOOGLE_CLOUD_PLATFORM_PROJECT_ID \
   --stagingLocation=gs://YOUR_BUCKET/dataflow-staging \
+  --tempLocation=gs://YOUR_BUCKET/dataflow-staging \
```

Comment:
The dataflow documentation says the tempLocation flag defaults to the value of stagingLocation, so this is a no-op.

  https://cloud.google.com/dataflow/pipelines/specifying-exec-params

Why add it?

Reply:
Sorry I guess this doesn't answer my question - is the dataflow documentation wrong then?  I see no mention in there of cases where you need that flag.

What error do you get running locally?  (I can try it but it will take some time to get to it)

If it's the case that the dataflow docs are wrong, we should at least file a bug with them (and update this commit message to explain this flag is in fact required when running locally).

Reply:
Apologies for being unclear.

The error when running locally is `java.lang.IllegalArgumentException: BigQueryIO.Write needs a GCS temp location to store temp files.`

Reply:
OK, thanks